stdlib.sh

```diff
@@ -586,10 +770,12 @@ function __STDLIB_API_1_std::log() { # {{{
 	# We don't care whether std_LOGFILE exists, but we do care whether it's
 	# set...
 	[[ -n "${std_LOGFILE:-}" && "${std_LOGFILE}" != "syslog" ]] \
-		&& output "${message}" >>"${std_LOGFILE}" 2>&1
+		&& { [[ ! -n "${outputFlag}" ]] && output "${message}" >>"${std_LOGFILE}" 2>&1; [[ -n "${outputFlag}" ]] && output "${outputFlag}" "${message}" >>"${std_LOGFILE}" 2>&1; }
 
 	if (( std_DEBUG )); then
-		__STDLIB_API_1_std::wrap "${prefix}" "${data}"
+		## 'output' adds a trailing white space when ${outputFlag} is empty...
+		(( !std_WRAPLOG )) && { [[ ! -n "${outputFlag}" ]] && output "${prefix} ${data}"; [[ -n "${outputFlag}" ]] && output "${outputFlag}" "${prefix} ${data}"; }
+		(( std_WRAPLOG )) && __STDLIB_API_1_std::wrap "${param_logFlag}" "${prefix}" "${data}"
```

Comment:
Yes, while trying to emphasise std_WRAPLOG I ended up with a bit of overcooked spaghetti code...